frontend/app/src/lib/api/generated/Api.ts

```diff
@@ -1732,6 +1732,10 @@ export class Api<SecurityDataType = unknown> extends HttpClient<SecurityDataType
        * @maxLength 36
        */
       workflowId?: string;
+      /** The workflow name to get runs for. */
+      workflowName?: string;
```

Comment:
what happens if both name and id are set?

Reply:
I was thinking about that. it's harmless I think, but there's an argument to be made that maybe only one of the two should be set? idk, it seems fine to me since they're both optional and all of these params are `AND` clauses so I think it should be pretty clear. WDYT?

Reply:
agreed, its fine